MapboxDirections/MBRouteOptions.swift

```diff
@@ -119,9 +155,21 @@ open class RouteOptions: DirectionsOptions {
             URLQueryItem(name: "alternatives", value: String(includesAlternativeRoutes)),
             URLQueryItem(name: "continue_straight", value: String(!allowsUTurnAtWaypoint))
         ])
-
-        if includesExitRoundaboutManeuver {
-            queryItems.append(URLQueryItem(name: "roundabout_exits", value: String(includesExitRoundaboutManeuver)))
+        
+        switch profileIdentifier {
+        case .automobile, .automobileAvoidingTraffic:
+            if includesExitRoundaboutManeuver {
+                queryItems.append(URLQueryItem(name: "roundabout_exits", value: String(includesExitRoundaboutManeuver)))
```

Comment:
All profiles support this parameter, and roundabouts do occasionally occur along bike/hike trails. Better that this library pass through the parameter so that the navigation SDK can assume its presence.